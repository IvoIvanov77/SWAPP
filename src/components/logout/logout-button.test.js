import React from 'react';
import { shallow } from 'enzyme';
import { MockedProvider } from '@apollo/react-testing';

import LogoutButton from './logout-button.container';
it('should render LogoutButton component', () => {
  expect(
    shallow(
      <MockedProvider>
        <LogoutButton />
      </MockedProvider>,
    ),
  ).toMatchSnapshot();
});
