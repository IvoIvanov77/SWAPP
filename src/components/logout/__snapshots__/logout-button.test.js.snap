// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render LogoutButton component 1`] = `ShallowWrapper {}`;
